src/metabase/public_settings/premium_features.clj

```diff
@@ -236,13 +236,32 @@
   []
   (boolean (seq (token-features))))
 
+(def ^:dynamic *premium-feature-overrides*
+  "Dynamic var holding a set of tokens which are temporarily considered to be enabled, even if the user's token does
+  not have that feature.
+
+  This allows eg. `:audit-app` functionality to use `:serialization` internally, even if the token only has
+  `:audit-app`.
+
+  Don't touch this directly - prefer to use [[with-premium-feature-overrides]]."
+  #{})
+
+(defmacro with-premium-feature-overrides
```

Comment:
Are you re-adding this intentionally? Don't see it being used

Reply:
Whoops, no, that should have been dropped. I guess that's a rebase ghost. I'll clear it out.